src/realm/util/file.hpp

```diff
@@ -1063,6 +1064,7 @@ inline bool File::try_lock_shared()
 inline File::MapBase::MapBase() noexcept
 {
     m_addr = nullptr;
+    m_fd = 0;
```

Comment:
Seems like you never read the 0 value from anywhere or do decisions based on it. So it's better to leave m_fd uninitialized becasue that will enable our debugging tools (Valgrind, Visual Studio in debug mode, etc) to catch any future cases where we by accident forget to assign it with a real value.

Reply:
Ok good idea, fixed.